winui/AvatarView/content-types.md

```diff
@@ -7,4 +7,305 @@ control: AvatarView
 documentation: ug
 ---
 
-# Content Types in WinUI AvatarView
\ No newline at end of file
+# Content Types in WinUI AvatarView
+
+## Content Types
+
```

Comment:
@MadhumithanJayapathi Remove the heading ## Content Types and explain what are the content types available.